files/k8s-resource/calico-all.yaml

```diff
@@ -399,15 +489,26 @@ spec:
             # Mount in the etcd TLS secrets.
             - mountPath: /calico-secrets
               name: etcd-certs
+          securityContext:
+            runAsUser: 0
```

Comment:
What does it change to need it now?

Reply:
Yulia made this change. I assume it was necessary to make it run, but I don't know for sure. Interestingly, in 3.18 there was this change so it would run as non-root https://github.com/projectcalico/kube-controllers/pull/565 I can try to revert it.

Reply:
Changes I did were documented here: https://github.com/giantswarm/giantswarm/issues/17954#issuecomment-898328797